pages/HomeScreen.tsx

```diff
@@ -51,7 +49,7 @@ import CustomHandle from "../components/CustomHandle";
 
 // Root component
 
-export function HomeScreen({ navigation }) {
+export function HomeScreen() {
```

Comment:
Why not the arrow function with type?

Something like

![image](https://user-images.githubusercontent.com/69431456/129467547-187e6d81-7f22-4970-abd2-b7e45dc7cf82.png)